modules/ducktests/tests/ignitetest/services/utils/control_utility.py

```diff
@@ -37,27 +39,11 @@ class ControlUtility:
     BASE_COMMAND = "control.sh"
 
     # pylint: disable=R0913
-    def __init__(self, cluster,
-                 key_store_jks: str = None, key_store_password: str = DEFAULT_PASSWORD,
-                 trust_store_jks: str = DEFAULT_TRUSTSTORE, trust_store_password: str = DEFAULT_PASSWORD):
+    def __init__(self, cluster, **kwargs):
         self._cluster = cluster
         self.logger = cluster.context.logger
-
-        if cluster.context.globals.get("use_ssl", False):
-            admin_dict = cluster.globals.get("admin", dict())
-
-            self.key_store_path = admin_dict.get("key_store_path",
-                                                 self.jks_path(admin_dict.get('key_store_jks', DEFAULT_ADMIN_KEYSTORE)))
-            self.key_store_password = admin_dict.get('key_store_password', DEFAULT_PASSWORD)
-            self.trust_store_path = admin_dict.get("trust_store_path",
-                                                   self.jks_path(admin_dict.get('trust_store_jks', DEFAULT_TRUSTSTORE)))
-            self.trust_store_password = admin_dict.get('trust_store_password', DEFAULT_PASSWORD)
-
-        elif key_store_jks is not None:
-            self.key_store_path = self.jks_path(key_store_jks)
-            self.key_store_password = key_store_password
-            self.trust_store_path = self.jks_path(trust_store_jks)
-            self.trust_store_password = trust_store_password
+        self.ssl_context = self._parse_ssl_params("admin", cluster.context.globals, **kwargs)
+        self.creds = self._parse_creds("admin", cluster.context.globals, **kwargs)
```

Comment:
just use
```
self.username, self.password = self._parse_creds()
```

Reply:
username and password are always used together, i don't see any good reason to separate them?
And we have 2 another passwords here (for keystore and for truststore)